_rules/table-headers-attribute-refer-to-data-cells-a25f45.md

```diff
@@ -0,0 +1,295 @@
+---
+id: a25f45
+name: Headers attribute specified on a cell refers to cells in the same table element
+rule_type: atomic
+description: |
+  This rule checks that the headers attribute on a cell refer to other cells in the same `table` element with a semantic role of `columnheader` or `rowheader`.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgements:
+  authors:
+    - Jey Nandakumar
+    - Audrey Maniez
+---
+
+## Applicability
+
+This rule applies to any `headers` attribute specified on a [`cell`](https://www.w3.org/TR/html52/tabular-data.html#cell) within a `table` element, where the `table` element is [visible](#visible) and [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+## Expectation
+
+Each target attribute is [a set of space separated IDs](https://www.w3.org/TR/html50/infrastructure.html#set-of-space-separated-tokens), each of which is the ID of an element, that:
+
+1. has a [semantic role](#semantic-role) of [columnheader](https://www.w3.org/TR/html-aria/#index-aria-columnheader) or [rowheader](https://www.w3.org/TR/html-aria/#index-aria-rowheader), and
```

Comment:
The rule is not specific about what the element is or isn't, only that it has an expected semantic role and is part of the same table. 

As an atomic rule checking a particular thing, a fail scenario would be inaccessible, while a pass scenario would be part of a wider set of checks for a SC. So I'm not seeing how the comment is still relevant.

Reply:
That would be a problem. Wilco states that this should not be required because it will fail situations that aren't actually a WCAG issue (VoiceOver not reading the headers correctly would not constitute a WCAG failure per se).